dist/test_data.py

```diff
@@ -220,7 +220,10 @@ def __ge__(self, other):
                             How long the insertions will occur for.''')]),
     'example_test' : Method(test_config),
     'hs_cleanup' : Method(test_config),
-    'search_near_01' : Method(test_config),
+    'search_near_01' : Method(test_config + [
+                        Config("search_near_threads", 10, r'''
+                            Config that specify the number of search near threads that are
```

Comment:
typo `specifies`

Reply:
Can we just say something along those lines `Number of threads that execute search near calls`